README.md

```diff
@@ -93,9 +93,9 @@ Setting Up dummy API KEY
 3. Paste the following code inside .env file.
 
 ```js
-REACT_APP_AIRTABLE_API_KEY = keyl7IrslG0riVZjW;
-REACT_APP_TEAM_BASE = appeFoa7ggqSOwivF;
-REACT_APP_EVENT_BASE = appC2QPeeuXQPAxgr;
+REACT_APP_AIRTABLE_API_KEY=keyl7IrslG0riVZjW
```

Comment:
Do we need this? @imabp

Reply:
Yes @vinitshahdeo , these are the dummy keys we have set for the contributors, which help them to feel how we are communicating with the airtable servers and also is a necessity to setup the development environment.

It's not a matter of security, as the key set is completely dummy and comes entirely from a different airtable account set for no use case.

Once we are complete with the issue #225 of integrating the Azure API using the endpoints in this repository [here](https://github.com/Tech-Phantoms/api), We will remove these dummy keys.